magellan-library/build.gradle

```diff
@@ -23,6 +23,11 @@ android {
     setSourceCompatibility(JavaVersion.VERSION_1_8)
     setTargetCompatibility(JavaVersion.VERSION_1_8)
   }
+
+  kotlinOptions {
```

Comment:
Need this for supporting `@JvmDefault`

Reply:
We should include this in the breaking changes list, though I doubt it'll affect many people

Reply:
Yup!